src/core/.ocamlformat

```diff
@@ -0,0 +1,10 @@
+version=0.18.0
+wrap-fun-args=false
+let-binding-spacing=compact
+field-space=loose
+sequence-style=separator
+doc-comments=before
+margin=80
+module-item-spacing=sparse
+parens-tuple=always
+parens-tuple-patterns=always
```

Comment:
Please configure your editor/Git to always put an empty line at the end of all files 😅 See e.g. https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

Reply:
My editor is supposed to do it, I must have removed it by mistake :/